WORKSPACE

```diff
@@ -1,9 +1,10 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
-   name = "rules_foreign_cc",
-   strip_prefix = "rules_foreign_cc-master",
-   url = "https://github.com/bazelbuild/rules_foreign_cc/archive/master.zip",
+    name = "rules_foreign_cc",
+    strip_prefix = "rules_foreign_cc-c3d5405cbc570257e7c9f75f902fab42241e6a53",
+    url = "https://github.com/bazelbuild/rules_foreign_cc/archive/c3d5405.zip",
```

Comment:
Can you add a comment like:

# Using an explicit commit as the project does not use tags for versioning.

just before the url?

Otherwise, a year from now when we go update the dependencies we'll be like 'why did we stick to a specific commit? was there a bugfix we had to worry about?'